tests/unit/test_db_sync.py

```diff
@@ -310,5 +311,35 @@ def test_record_primary_key_string(self, query_patch):
         stream_schema_message['key_properties'] = ['invalid_col']
         dbsync = db_sync.DbSync(minimal_config, stream_schema_message)
         with self.assertRaisesRegex(PrimaryKeyNotFoundException,
-                                    "Cannot find \['invalid_col'\] primary key\(s\) in record\. Available fields: \['id', 'c_str'\]"):
+                                    r"Cannot find \['invalid_col'\] primary key\(s\) in record\. Available fields: \['id', 'c_str'\]"):
             dbsync.record_primary_key_string({'id': 123, 'c_str': 'xyz'})
+
+    @patch('target_snowflake.db_sync.DbSync.query')
```

Comment:
I removed this test and noticed test coverage is 62.17% which is greater than 62% yet! but it is good to have this test and keep more than 65% test coverage.

Reply:
:+1: